src/functionalTest/resources/scenarios/RIA-8303-aip-appellant-notifications-refund-decision-approved.json

```diff
@@ -0,0 +1,106 @@
+{
+  "description": "RIA-8303 AIP Appellant notifications: refund decision approved",
+  "launchDarklyKey": "dlrm-refund-feature-flag:true",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 8303,
+      "eventId": "recordRemissionDecision",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "aip-minimal-appeal-submitted.json",
+        "replacements": {
+          "remissionDecision": "approved",
+          "remittedAmount": "140",
+          "isDlrmFeeRefundEnabled": "Yes",
+          "isLateRemissionRequest": "Yes",
+          "subscriptions": [
+            {
+              "id": "1",
+              "value": {
+                "subscriber": "appellant",
+                "email": "{$TEST_CITIZEN_USERNAME}",
+                "wantsEmail": "Yes",
+                "mobileNumber": "{$TEST_CITIZEN_MOBILE}",
+                "wantsSms": "Yes"
+              }
+            }
+          ],
+          "notificationsSent": []
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "aip-minimal-appeal-submitted.json",
+      "replacements": {
+        "remissionDecision": "approved",
+        "remittedAmount": "140",
+        "subscriptions": [
+          {
+            "id": "1",
+            "value": {
+              "subscriber": "appellant",
+              "email": "{$TEST_CITIZEN_USERNAME}",
+              "wantsEmail": "Yes",
+              "mobileNumber": "{$TEST_CITIZEN_MOBILE}",
+              "wantsSms": "Yes"
+            }
+          }
+        ],
+        "notificationsSent": [
+          {
+            "id": "8303_REMISSION_DECISION_APPROVED_ADMIN_OFFICER",
```

Comment:
Let's wait for Ben's confirmation, then we can remove the admin email. Remember to add !isAipJourney on the admin config.